copier.yaml

```diff
@@ -113,3 +129,52 @@ default_py:
       value: '3.11'
   help: 'Choose the default Python version for development, documentation generation, and package build:'
   type: str
+
+copyright_license:
+  type: str
+  help: Your project's license. If you have no idea how to choose one, please see https://choosealicense.com/.
+  default: MIT License
+  choices:
+    - Academic Free License v3.0
```

Comment:
1. Align it with [PyPI classifiers](https://pypi.org/classifiers/). For example, `Academic Free License v3.0` should be `Academic Free License (AFL)`, `Apache License 2.0` should be `Apache Software License`.
2. I may suggest to only use **those most popular ones** and you can find them (8 in total) on the [choosealicense](https://choosealicense.com/licenses/)

Reply:
Yep, there are too many licenses so that i feel a little cubersome management.